src/components/DisplayMetrics.js

```diff
@@ -68,7 +72,15 @@ class DisplayMetrics extends React.Component {
   render() {
     const numberOfClicks = this.state.clicks;
     const numberOfImpresions = this.state.impressions;
-    const selectSuggestions = _.uniqWith(this.state.suggestions, _.isEqual);
+    const uniqueCampaign = _.uniqWith(
+      this.state.suggestionsForCampaign,
+      _.isEqual
+    );
+    const uniqeChannel = _.uniqWith(
+      this.state.suggestionsForChannel,
+      _.isEqual
+    );
```

Comment:
don't repeat youself DRY 

please extract this to a function (eg `uniqueObjects`) and place this in `src/utils`

Reply:
648eb83d51cb96ab87ac8c5d2f4df8a04c2c1336